kedro/extras/logging/__init__.py

```diff
@@ -1,7 +1,15 @@
 """
 This module contains a logging handler class which produces coloured logs.
 """
+import warnings
 
 from .color_logger import ColorHandler
 
 __all__ = ["ColorHandler"]
+
+warnings.simplefilter("default", DeprecationWarning)
+
+warnings.warn(
+    "Support for ColorHandler will be deprecated in Kedro 0.19.0.",
```

Comment:
```suggestion
    "`ColorHandler` will be removed in Kedro 0.19.0.",
```

This sounds nitpicky, especially because we aren't always very consistent about it, but I think we should try to be. Removal is not the same as deprecation: we _deprecate_ something now in 0.18.x to say "you shouldn't use this" and then it is _removed_ in 0.19.0.